include/f_core/os/c_task.h

```diff
@@ -79,6 +82,9 @@ class CTask {
     k_thread_stack_t* stack;
 
     std::vector<CRunnableTenant*> tenants;
+    wdt_timeout_cfg *wdtConfig = nullptr;
+    const device *watchdogDev = nullptr;
```

Comment:
why does a task need to hold onto the watchdog dev, shouldnt it be done using it after Initialize ends